src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In JavaScript, Which Web API used to insert a child node anywhere under the parent node?",
+    Answer: "insertBefore()",
+    Distractor1: "prepend()",
+    Distractor2: "append()",
+    Distractor3: "lookupPrefix()",
+    Explanation:
+      "The insertBefore() method of the Node interface inserts a node before a reference node as a child of a specified parent node.",
```

Comment:
This is a good explanation but we want to avoid copying from MDN docs. 

Please rewrite this in your own words.

Reply:
Yaa sure I will update the explanation.

Reply:
I updated the explanation @jdwilkin4